flutter-studio/src/io/flutter/profiler/FlutterStudioMonitorStageView.java

```diff
@@ -634,7 +639,10 @@ public void selectionCleared() {
       new RangedContinuousSeries("MemoryMax", getTimeline().getViewRange(), dataRanges, memoryMaxDataSeries);
     RangedContinuousSeries externalMemoryRange =
       new RangedContinuousSeries("MemoryExtern", getTimeline().getViewRange(), dataRanges, memoryExternalDataSeries);
+//    RangedContinuousSeries rssRange = new RangedContinuousSeries("RSS",
```

Comment:
add a quick comment on why this is commented out

Reply:
also I assume that "RSS" will show up in the UI. We'll need to use a longer name as users will think RSS means something completely different.